<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <node name="joy" pkg="joy" exec="joy_node"/>
    
    <node name="road_bot_bridge" pkg="road_bot" exec="road_bot_bridge"/>
    
    <!--node name="camera_bridge" pkg="road_bot" exec="camera_bridge"/-->
    
    <!--node name="image_view" pkg="image_view" exec="image_view" output="screen">
        <param name="autosize" value="true" />
        <remap from="image" to="rpi_image_raw" />
    </node-->
</launch>
